test/FsAutoComplete.Tests.Lsp/Tests.fs

```diff
@@ -1029,6 +1029,20 @@ let fakeInteropTests =
         | Core.Result.Error errors ->
           failwithf "Errors while parsing script %s: %A" (Path.Combine(rootPath, scriptName)) errors
         ))
+    ftestCase "can determine fake targets list for a fake script" (serverTest (fun (server, events, rootPath, scriptName) ->
+        do server.TextDocumentDidOpen { TextDocument = loadDocument (Path.Combine(rootPath, scriptName)) } |> Async.RunSynchronously
+        match waitForParseResultsForFile scriptName events with
+        | Ok () ->
+          match server.FakeTargets({ FileName = Path.Combine(rootPath, scriptName)
+                                     FakeContext = { DotNetRuntime = "netcoreapp2.1" } }) |> Async.RunSynchronously with
```

Comment:
I took a brief look at this.
The test is wrong: The variable `DotNetRuntime` should be the path to the `dotnet` binary (`dotnet.exe` on windows).
Maybe some internal FSAutocomplete Variables changed in their semantic?

Is there some change that comes to mind?

Reply:
What we probably can do (in addition to fixing this variable) is patching it to fail with a better error message.

Reply:
I tweaked the test a bit to provide the path to dotnet, and I've got better error messages now 👍  Now I need to figure out why my sample script fails with a MissingMethodException for `Async.Parallel` in FSharp.Core.

Reply:
For me the test went green. But the question is why ionide itself is still failing with this error?

Reply:
quite possibly, yeah. after we merge this test to pin down functionality I'd look at ensuring Ionide sends the correct thing